demos/data-lakehouse-iceberg-trino-spark/create-spark-ingestion-job.yaml

```diff
@@ -144,13 +144,13 @@ data:
         stackable.tech/vendor: Stackable
     spec:
       sparkImage:
-        productVersion: 3.5.6
+        productVersion: 3.5.7
       mode: cluster
       mainApplicationFile: local:///stackable/spark/jobs/spark-ingest-into-lakehouse.py
       deps:
         packages:
           - org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:1.8.1
```

Comment:
I think it would be great if we could bump the demos to Spark 4.0.1 and Iceberg 1.10.0